Notes_and_Examples/5_CurrentFlow/5_currentDirection.xml

```diff
@@ -0,0 +1,108 @@
+<problem display_name="webGLDemo">
```

Comment:
@e-kolpakov Great catch! I generally changes these in the "settings" tab in studio, but better to do when I start coding.